invenio_app_ils/records/resolver/jsonresolver/document_item.py

```diff
@@ -32,16 +34,37 @@ def items_resolver(document_pid):
                 "medium": item.get("medium"),
                 "status": item.get("status"),
                 "description": item.get("description"),
+                "shelf": item.get("shelf"),
+                "internal_location": {
+                    "location": {
+                        "name": item.get("internal_location")
+                                    .get("location").get("name")
+                    },
+                    "name": item.get("internal_location").get("name")
+                }
             }
             if circulation:
                 include_circulation_keys = ['state']
                 obj["circulation"] = {}
                 for key in include_circulation_keys:
                     obj["circulation"][key] = circulation.get(key)
             items.append(obj)
+
+            # grouping by location (can circulate and not on loan)
+            location_name = item["internal_location"]["location"]["name"]
+            internal_location_name = item["internal_location"]["name"]
+            if "circulation" not in obj and can_item_circulate(item["pid"]):
```

Comment:
I am wondering: do you think it is better to display all of them, with its circ. status, e.g. ON LOAN, or hide the ones on loan? (it looks like here they are not added right?